src/output.rs

```diff
@@ -46,7 +46,7 @@ pub fn get_target_output_writable(output_file: Option<&str>) -> Box<dyn Write> {
                 )
             }
             Box::new(File::create(output).unwrap_or_else(|_| {
-                panic!("Cannot create the file {} to dump coverage data", filename)
+                panic!("Cannot create the file {} to dump coverage data. Check if parent directory exists.", filename)
```

Comment:
This error isn't only printed when the parent directory doesn't exist. Instead of asking the user to check, grcov can check first if the parent directory exists and panic if it doesn't with a clearer error message "Cannot create path/to/file to dump coverage, as path/to/ doesn't exist".

Reply:
Ah great. I was thinking about that. I'm changing it!

Reply:
The code I wrote looks very dirty. How would you ameliorate it?

I also tried: 
```rust
output.pop();
if !output.exists() {
    panic!(
        "Cannot create {} to dump coverage data, as {} doesn't exist",
        filename, output.display()
    )
}
```
but we need to make output mutable

and
```rust
match output.parent() {
                    Some(parent_path) => {
                        if !parent_path.exists() {
                            panic!(
                                "Cannot create {} to dump coverage data, as {} doesn't exist",
                                filename, parent_path.display()
                            )
                        }
                    },
                    None => {}
                };
```

and 

```rust
let parent = output.parent().unwrap();
                if !parent.exists() {
                    panic!(
                        "Cannot create {} to dump coverage data, as {} doesn't exist",
                        filename, parent.display()
                    )
                }
```